fastparquet/core.py

```diff
@@ -150,8 +151,12 @@ def read_col(column, schema_helper, infile, use_cat=False):
     use_cat: bool (False)
         If this column is encoded throughout with dict encoding, give back
         a pandas categorical column; otherwise, decode to values
+    grab_dict: bool (False)
+        Short-cut mode to return the dictionary values only - skips the actual
+        data.
```

Comment:
This sort of "tack a keyword onto an existing function to provide a distinct but related function" feels like an anti-pattern.  It may be worth it though.  I don't have enough experience with the codebase to tell.